ui/js/page/filePage/view.jsx

```diff
@@ -119,7 +120,11 @@ class FilePage extends React.PureComponent {
               </div>
             </div>
             <div className="card__content card__subtext card__subtext card__subtext--allow-newlines">
-              {metadata && metadata.description}
+              <ReactMarkdown
+                source={(metadata && metadata.description) || ""}
+                escapeHtml={true}
+                disallowedTypes={["Heading", "HtmlInline", "HtmlBlock"]}
```

Comment:
I don't see that in the list of possible types to disallow.

![2017-07-12 at 14 14](https://user-images.githubusercontent.com/20863631/28106079-97579768-670c-11e7-9af8-4fa65b2acf88.jpg)